src/lib/eliom_request_info.server.mli

```diff
@@ -233,10 +233,6 @@ val get_persistent_cookies :
 val get_data_cookies :
   unit -> string Eliom_common.Full_state_name_table.t
 
-
-(** Returns the http error code of the request before Eliom was called *)
-val get_previous_extension_error_code :unit -> int
```

Comment:
Are you sure that nobody will need that?

Reply:
Oh, sorry. For some reason I thought that the function was not visible to the outside world. Fixed.